gluster-exporter/metric_volume.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	volumeLabels = []string{
+		"volume",
+	}
+
+	glusterVolumeTotalCount = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "gluster",
+			Name:      "volume_total_count",
+			Help:      "Total no of volumes",
+		},
+		[]string{},
+	)
+	glusterVolumeStartedCount = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "gluster",
+			Name:      "volume_started_count",
+			Help:      "Total no of started volumes",
+		},
+		[]string{},
+	)
+	glusterVolumeStoppedCount = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "gluster",
+			Name:      "volume_stopped_count",
```

Comment:
This metric can be derived by rule if required(`total - started`)

Reply:
Ack. Will remove this from here.